API/SearchService/src/util.py

```diff
@@ -24,6 +24,11 @@ class MatchRule(int, Enum):
     Or = 3
 
 
+class ScenarioSource(int, Enum):
+    SAMPLE_REPO = 1
+    DOC_CRAWLER = 2
```

Comment:
It is recommended to reuse `ScenarioSourceType` instead of writing a similar enumeration

Reply:
> It is recommended to reuse `ScenarioSourceType` instead of writing a similar enumeration

Done. I've extracted `ScenarioSourceType` to a common scope and reused it.